src/urllib3/exceptions.py

```diff
@@ -22,7 +22,7 @@ class PoolError(HTTPError):
 
     def __init__(self, pool, message):
         self.pool = pool
-        HTTPError.__init__(self, "%s: %s" % (pool, message))
+        HTTPError.__init__(self, "{}: {}".format(pool, message))
```

Comment:
All of the changes from `%` to `.format()` aren't great because `%` is more performant and the same level of readable. I don't know if `pyupgrade` is configurable in this way?